static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -76,6 +77,33 @@ export function getCoverageColors(
   });
 }
 
+export function getCoverageAnalytics(
+  lineColors: Array<Color | 'transparent'>,
+  primaryLineIndex: number,
+  organization: Organization | null
+): void {
+  let primaryLineCoverageChecked = false;
+  let surroundingLinesCovered = true;
+  for (const index in lineColors) {
+    if (parseInt(index, 10) === primaryLineIndex) {
+      trackAdvancedAnalyticsEvent('issue_details.codecov_primary_line_coverage_shown', {
+        organization,
+        success: lineColors[index] !== 'transparent',
+      });
+      primaryLineCoverageChecked = true;
+    } else if (lineColors[index] === 'transparent') {
+      surroundingLinesCovered = false;
```

Comment:
If any of the surrounding lines are transparent, the surrounding lines are not covered